docs/regmap/adi_regmap_spi_engine.txt

```diff
@@ -241,6 +248,21 @@ ENDFIELD
 ############################################################################################
 ############################################################################################
 
+REG
+0x31
+OFFLOAD_SYNC_ID
+ENDREG
+
+FIELD
+[31:0] 0x00000000
+OFFLOAD_SYNC_ID
+RO
+Offload Sync ID.
+ENDFIELD
+
+############################################################################################
+############################################################################################
+
 REG
 0x34
 CMD_FIFO_ROOM
```

Comment:
replace 0x???????? with ((2^CMD_FIFO_ADDRESS_WIDTH) -1)  if possible

for SDO_FIFO_ROOM
replace 0x???????? with ((2^SDO_FIFO_ADDRESS_WIDTH) -1)  if possible

replace 0x00000000 with 0x0
FIELD
[31:0] 0x00000000
SDI_FIFO_LEVEL

Reply:
Done.